### PR TITLE
Role for Nart team to gain port-forward and access to RDS instances

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/07-rbac-nart-team.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nart-team-portforward
+  namespace: offender-categorisation-dev
+subjects:
+  - kind: Group
+    name: "github:dps-nart-team-access"
+    apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/07-rbac-nart-team.yaml]: Can you change it to kind: Role please

Comment (vijay-veeranki-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/07-rbac-nart-team.yaml]: For other two namespaces as well please

Comment (mattops) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/07-rbac-nart-team.yaml]: Good spot. Yes I'll fix.

Comment (mattops) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/07-rbac-nart-team.yaml]: Now fixed on all 3 namespaces in this PR.

**Pull request merged**